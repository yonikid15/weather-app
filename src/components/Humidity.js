import React from 'react';

const Humidity = ({ humidity }) => {
  return (
    <div>
      <h3>
          <span>Humidity: </span>
          <span>{humidity}</span>
      </h3>
    </div>
  );
};

export default Humidity;