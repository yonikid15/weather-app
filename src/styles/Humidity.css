.humidity {
  grid-column: 4 / 6;
  grid-row: 4;
  background-color: rgba(0, 0, 0, 0.233);
  border-radius: 10px;
  text-align: center;
  box-shadow: 0 19px 38px rgba(0, 0, 0, 0.3), 0 15px 12px rgba(0, 0, 0, 0.22);
  display: flex;
  justify-content: center;
  align-content: center;

  &__title {
    font-weight: 500;
    font-size: 25px;
  }

  &__icon {
    padding: 10px;
    font-size: 25px;
  }
}