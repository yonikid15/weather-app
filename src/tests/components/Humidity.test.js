import React from 'react';
import { shallow } from 'enzyme';
import Humidity from '../../components/Humidity';

test('should render Humidity correctly', () => {
  const wrapper = shallow( <Humidity hmidity={80} /> );
  expect( wrapper ).toMatchSnapshot();
});