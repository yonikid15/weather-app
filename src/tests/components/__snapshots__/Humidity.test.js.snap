// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Humidity correctly 1`] = `
<div
  className="humidity"
>
  <h3
    className="humidity__title"
  >
    <i
      className="fas fa-tint humidity__icon"
    />
    <span />
  </h3>
</div>
`;
